Failure in rebinning due to scipy internals change

The failure is during a call to stingray.utils._root_squared_mean:
```
     def _root_squared_mean(array):
>       return np.sqrt(np.sum(array ** 2)) / len(array)
E       TypeError: unsupported operand type(s) for ** or pow(): 'list' and 'int'

```
This should be an easy fix: add an `array=np.asarray(array)` before the `return` statement

--- a/stingray/utils.py
+++ b/stingray/utils.py
@@ -102,7 +102,7 @@ __all__ = ['simon', 'rebin_data', 'rebin_data_log', 'look_for_array_in_array',
 
 def _root_squared_mean(array):
     array = np.asarray(array)
-    return np.sqrt(np.sum(array ** 2)) / len(array)
+    return np.sqrt(np.sum(array ** 2)) / array.size
 
 
 def simon(message, **kwargs):

--- a/stingray/utils.py
+++ b/stingray/utils.py
@@ -101,6 +101,7 @@ __all__ = ['simon', 'rebin_data', 'rebin_data_log', 'look_for_array_in_array',
 
 
 def _root_squared_mean(array):
+    array = np.asarray(array)
     return np.sqrt(np.sum(array ** 2)) / len(array)
 
 

